Viewer/src/net/relinc/viewer/GUI/ChartsGUI.java

```diff
@@ -53,9 +118,9 @@ public ChartsGUI(HomeController hc)
 		XAxis.setLabel(xlabel + " " + xUnits);
 		YAxis.setLabel(yLabel + " " + yUnits);
 
-		LineChartWithMarkers<Number, Number> chart = new LineChartWithMarkers<>(XAxis, YAxis, chartDataType.TIME, chartDataType.STRESS);
+		LineChartWithMarkers<Number, Number> chart = new LineChartWithMarkers<>(XAxis, YAxis, chartDataType.TIME, chartDataType.STRESS, chartType.STRESSTIME);
+		chart.chartWithPreferences(preference, "Stress Vs Time");
```

Comment:
Instead of passing in `"Stress Vs Time"`, can we just call `chart.chartWithPreferences(preference)`

and then the implementation of that method is an overload method such as:

```java
public void chartWithPreferences(preference) {
    chartWithPreferences(preference, getDefaultTitle())
}
```

Does that make sense? Just to delete some code/configuration of the default titles

Reply:
why, pray tell?

Reply:
scratch that. i smell what you are stepping in.